python/packages/sdk/serverless_sdk/__init__.py

```diff
@@ -46,5 +49,14 @@ def _create_trace_span(
     ) -> trace.TraceSpan:
         return trace.TraceSpan(name, input, output, start_time, tags)
 
+    def capture_error(self, error, **kwargs) -> CapturedEvent:
+        try:
+            _error = create_error_captured_event(error, **kwargs)
+            self._captured_events.append(_error)
+            return _error
+        except Exception:
+            # TODO report error
```

Comment:
What's the plan of addressing that `TODO`?

Reply:
I've implemented that bit now 👍